cranelift-frontend/src/switch.rs

```diff
@@ -9,6 +9,36 @@ type EntryIndex = u64;
 
 /// Unlike with `br_table`, `Switch` cases may be sparse or non-0-based.
 /// They emit efficient code using branches, jump tables, or a combination of both.
+///
+/// # Example
+///
+/// ```rust
+/// # use cranelift_codegen::ir::types::*;
+/// # use cranelift_codegen::ir::{ExternalName, Function, Signature};
+/// # use cranelift_codegen::isa::CallConv;
+/// # use cranelift_frontend::{FunctionBuilder, FunctionBuilderContext, Switch};
```

Comment:
CI broke, saying you need `use cranelift_codegen::ir::builder::InstBuilder;` here too to be able to use `builder.ins().iconst` below.